typings/internalBinding/fs.d.ts

```diff
@@ -292,6 +292,7 @@ export interface FsBinding {
   writeBuffer: typeof InternalFSBinding.writeBuffer;
   writeBuffers: typeof InternalFSBinding.writeBuffers;
   writeString: typeof InternalFSBinding.writeString;
+  writeFileUtf8: typeof InternalFSBinding.writeFileUtf8;
```

Comment:
```suggestion
  writeFileUtf8: typeof InternalFSBinding.writeFileUtf8;
  writeString: typeof InternalFSBinding.writeString;
```
nit: methods seem to be arranged in alphabetical order.

Reply:
Done. Thank your comment!